src/bittybuzz/include/bittybuzz/BittyBuzzUserFunctions.h

```diff
@@ -12,18 +12,65 @@ namespace BittyBuzzUserFunctions {
 
     /**
      *@brief Logs a number value. Use it for sanity checks, it has no real production values
-     *
      *@details This closure expects one integer parameter, the number to log. Returns nothing
      * */
     void logInt();
 
     /**
      *@brief Logs a string value
-     *
      *@details This closure expects one stringId parameter, the string to log. Returns nothing
      * */
     void logString();
 
+    /**
```

Comment:
The `BittyBuzzUserFunctions` namespace contains some user-defined functions right? Does that mean that type checking will have to be managed and implemented by the user or is this feature a part of the "platform"?

Reply:
UserFunctions are the function that we make available to the user. In other words, we add functions to the buzz VM, this is those function and the user can use them in their buzz script.